repo/packages/C/crate/0/marathon.json

```diff
@@ -0,0 +1,26 @@
+{
+    "id": "/crate/{{crate/cluster-name}}",
+    "instances": 1,
+    "cpus": 0.25,
+    "mem": 128,
+    "ports": [{{framework/api-port}}],
+    "env": {
+{{#framework/auth/enabled}}
+      "MESOS_AUTHENTICATE": "true",
+      "DEFAULT_PRINCIAL": "{{framework/auth/princial}}",
+      "DEFAULT_SECRET": "{{framework/auth/secret}}"
+{{/framework/auth/enabled}}
+    },
+    "uris": [
+        "https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz",
+        "https://cdn.crate.io/downloads/releases/crate-mesos-0.1.0-SNAPSHOT-cec0ad0.tar.gz"
```

Comment:
Not true.

At this time (since we're not doing binary asset management) having URIs from a reliable location (such as S3, or a CDN) are allowed as one of the packaging options.

And URLs do not need to be configurable, this is how a framework "bakes" it's release.  Cassandra does this as well, and we don't want users changing the versions of things out from underneath us.

The reference to `https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz` is not allowed in this scenario though.  This is due to licensing restrictions with the jre. The way that Mesosphere is using it right now is Redistribution.  We are Redistributing the jre as part of our application (cassandra framework).  We are no hosting the jre for anyone to use, because then we would not longer be redistributing as part of our application, and would be in violation of the license agreement.

You will have to host your own copy of the jre (whether oracle or openJDK) and the URI should be updated to this new location.